element_calcium_imaging/imaging_no_curation.py

```diff
@@ -353,6 +351,7 @@ class ZDriftMetrics(dj.Computed):
     -> scan.Scan
     -> ZDriftParamSet
     ---
+    exceeds_threshold: boolean  # `True` if any value in z_drift > threshold from drift_params.
```

Comment:
Instead of a boolean like this, why not directly store a `longblob` of the frame indices of the frames exceeding the threshold (call it `bad_frames`)?